src/navigation/TabNavigator.tsx

```diff
@@ -7,6 +7,7 @@ import {
 import ThemeText from '@atb/components/text';
 import {LocationWithMetadata} from '@atb/favorites/types';
 import {usePreferenceItems} from '@atb/preferences';
+import * as Assistant_v2 from '@atb/screens/Assistant_v2/';
```

Comment:
Greit med default import her

```suggestion
import Assistant_v2 from '@atb/screens/Assistant_v2/';
```